walker_lake/Walker_Lake.ipynb

```diff
@@ -48,10 +48,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# arbitrarily choose a small memory limit (4GB) to stress the \n",
+    "# arbitrarily choose a small memory limit (4GB) to stress the\n",
     "# out of core processing infrastructure\n",
     "from dask.distributed import Client\n",
-    "client = Client(memory_limit=10e10, processes=False) # Note: was 6e9 \n",
+    "client = Client(memory_limit=10e10, n_workers=8, threads_per_worker=1) # Note: was 6e9 \n",
```

Comment:
Seems like these messages need updating; neither 10e10 nor 6e9 are 4GB, so please make the value match the message or the message match the value ("100GB"?).

Reply:
Also please add a comment above "client" saying that because Datashader uses parallel Numba for raster rendering, we tell Dask to use only a single thread on a given CPU to avoid contention.

Reply:
Addressed in ab5cd76

Reply:
Your second comment only just appeared for me. I'll add that description shortly.

Reply:
Comment added in a39c919. I think this is probably worth some markdown explanation as well tbh so I'll add that now.